doc/tutorials/New_Users.rst

```diff
@@ -0,0 +1,80 @@
+.. role:: html(raw)
+   :format: html
+
+.. _New_Users:
+
+Pennylane QuickStart
+=====================
+
+These step-by-step tutorials familiarize the user with the basics of PennyLane. You will be more confident to implement your own interesting problems in PennyLane after you are done with this QuickStart!  
```

Comment:
Might be best to avoid using the term 'user' in the tutorials section. Maybe something like this?
```suggestion
These step-by-step tutorials provided a gentle introduction to the basics of PennyLane. Once you have worked your way through the QuickStart tutorials, you will be ready to implement your own interesting QML problems in PennyLane!
```